sample/NCronJobSample/Jobs/PrintHelloWorldJob.cs

```diff
@@ -12,13 +12,14 @@ public Task RunAsync(JobExecutionContext context, CancellationToken token)
     {
         ArgumentNullException.ThrowIfNull(context);
 
-        LogMessage(context.Parameter);
+        if (!string.IsNullOrEmpty(context.Parameter as string))
```

Comment:
TBH: I would drop the check as we are sure that it is set.

Reply:
Not really, I added the check because the console just kept outputting `Message: [Null]`.
context.Parameter is a nullable object and was not being set in the sample test. As an example to users I didn't want this to confuse them if they were seeing this as well.
I do have a few ideas on how to pass more structured data, but I'll vet it out before submitting anything.

Reply:
Okay fair point - would it make sense just to add a parameter in this case?

Reply:
Well - there should be one:
```csharp
AddJob<PrintHelloWorldJob>(p => p.WithCronExpression("* * * * *").WithParameter("Hello from NCronJob"))
```

Reply:
Yes, but when working with nullable values I always have to assume that there is always a more-than-zero change that the value will be null. I think checking for null is a good defensive programming strategy.

Reply:
Absolutely - I, personally, just differ here in "is this code production or test/demo code".
If it is the latter and I know for certain that something is set, I will not go defensive.

Reply:
I got you. For test/demo code you want to keep it as concise as possible.